doc/changes/devel.rst

```diff
@@ -23,7 +23,8 @@ Version 1.7.dev0 (development)
 
 Enhancements
 ~~~~~~~~~~~~
-- None yet
+- Speed up export to .edf in :func:`mne.export.export_raw` (:gh:`12218` by :newcontrib:`Florian Hofer`)
```

Comment:
Should mention the switch from `EDFlib-Python` to `edfio` explicitly because people might need to `pip install` stuff to adjust